### ISSUE TITLE
Cannot export to draw.io. Browser console shows CORS error.

### ISSUE THREAD

**Issue by migscabral** [aws-solutions/workload-discovery-on-aws#233]
**Describe the bug**
Nothing happens when I export to draw.io. Browser console shows CORS errors.

**To Reproduce**

1. Go to Actions -> Export -> Draw.io
2. Nothing happens.
3. Open browser console, and see CORS errors

**Expected behavior**

Diagram is exported to draw.io

**Screenshots**

![Screen Shot 2021-12-02 at 10 11 10 AM](https://user-images.githubusercontent.com/1070620/144345996-f70e1651-8145-4afe-aa26-f50d5668ea5e.png)

**Browser (please complete the following information):**

- Name: chrome
- Version: 95.0.4638.69 (Official Build) (x86_64)

**Additional context**

Error from browser console:

```
Access to XMLHttpRequest at 'https://x3isczre30.execute-api.us-west-2.amazonaws.com/Prod//resources' from origin 'https://d1b7om70zj2v3b.cloudfront.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
...
x3isczre30.execute-api.us-west-2.amazonaws.com/Prod//resources:1 Failed to load resource: net::ERR_FAILED
```

**Comment by thebutler12** [comment]
Sorry for the delay getting back to you on this. Could you please take a look at the **DrawIOExportFunction** logs and see if there are any errors?

This may be related to this [issue](https://github.com/awslabs/aws-perspective/issues/232). We are working on a fix for this.

**Comment by chatii** [comment]
@thebutler12 

I have the same error on v1.1.4

DrawIOExportFunction error log

```
[ERROR] TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'
Traceback (most recent call last):
  File "/var/task/main.py", line 178, in handler
    xml_output = produce_xml_output(elements)
  File "/var/task/main.py", line 216, in produce_xml_output
    xml_object = elem.get_xml_object()
  File "/var/task/main.py", line 99, in get_xml_object
    'x': str(self.x),
  File "/var/task/main.py", line 67, in x
    children_points = list(
  File "/var/task/main.py", line 68, in <lambda>
    filter(None, map(lambda c: (c.center_x - 0.5*c.width), self.children)))
  File "/var/task/main.py", line 54, in width
    children_points = list(
  File "/var/task/main.py", line 55, in <lambda>
    filter(None, map(lambda c: (c.width*0.5 + c.center_x), self.children)))
  File "/var/task/main.py", line 54, in width
    children_points = list(
  File "/var/task/main.py", line 55, in <lambda>
    filter(None, map(lambda c: (c.width*0.5 + c.center_x), self.children)))
  File "/var/task/main.py", line 54, in width
    children_points = list(
  File "/var/task/main.py", line 55, in <lambda>
    filter(None, map(lambda c: (c.width*0.5 + c.center_x), self.children)))
```

**Comment by svozza** [comment]
This will be fixed in v2.1.0.

**Comment by morjoan** [comment]
Addressed in v2.1.0
